backend/app/main/service/user_service.py

```diff
@@ -96,8 +96,16 @@ def confirm_user_follow_request(data):
 
     return response_object, 201
 
-def get_a_user():
-    pass
+def get_a_user(public_id):
+    user = db.session.query(User.id, User.email, User.fb_id, User.name) \
+            .filter(User.id == public_id) \
+            .first()
+    response = user._asdict()
+    return response
+
+def get_user():
+    current_user = get_current_user()
```

Comment:
why can't you just return `get_current_user()`

Reply:
wait I mean `get_current_user()._as_dict()` or something like that

Reply:
I think we want the user details in the db, like name and email. get_current_user looks like a flask function so I don't think it returns that?